sbb/agora2mets.pl

```diff
@@ -0,0 +1,488 @@
+#!/opt/ActivePerl-5.16/bin/perl
```

Comment:
I use ActivePerl in both environments. So I don´t interfere with system perl.
I need install module like xml::LibXml and I can use the ppm from activeperl.
--
******************************************************************
Armin Möller,                     Administrator
Abt. IDM Ref. 2                 Tel +49 30 266 432231
Staatsbibliothek zu Berlin PK
Potsdamer Str. 33
10772 Berlin                         armin.moeller@sbb.spk-berlin.de<mailto:armin.moeller@sbb.spk-berlin.de>

Von: Stefan Weil [mailto:notifications@github.com]
Gesendet: Montag, 21. September 2015 17:56
An: goobi/goobi-contrib
Cc: Möller, Armin
Betreff: Re: [goobi-contrib] rdf-Format der Goobi-xml-Dateien in das neue mets-Format wandeln (#1)


In sbb/agora2mets.pl<https://github.com/goobi/goobi-contrib/pull/1#discussion_r39988000>:

> @@ -0,0 +1,488 @@

> +#!/opt/ActivePerl-5.16/bin/perl

ActivePerl is typical for Windows environment, but typical Linux distributions use /usr/bin/perl.

—
Reply to this email directly or view it on GitHub<https://github.com/goobi/goobi-contrib/pull/1/files#r39988000>.